src/pages/home_page/HomePage.jsx

```diff
@@ -0,0 +1,14 @@
+import i18next from "../../i18next";
+import FeatureSection from "./components/FeatureSection";
+import HiroSection from "./components/hiro_section/HiroSection";
```

Comment:
Peux tu changer `HiroSection` en `HeroSection`

Reply:
aeverthing is okay